imagick/magick_version.go

```diff
@@ -5,11 +5,26 @@
 package imagick
 
 /*
+#include <MagickCore/MagickCore.h>
 #include <MagickWand/MagickWand.h>
 */
 import "C"
 import "unsafe"
 
+// Returns the ImageMagick delegates as a string constant.
+func GetDelegates() (delegates string) {
```

Comment:
We don't need to use a named returned value here if we aren't actually using the allocated string.

Reply:
I pushed a fixup commit to change that. Or do you prefer to squash the commits together?